### DISCUSSION TITLE
🔥 Exciting #AI features are coming to Microcks! 🤖

### DISCUSSION THREAD

**Discussion by yada** [microcks/microcks#869] [Announcements]
AI Copilot 💡 Watch the early prototype in action! 
See:
https://www.linkedin.com/feed/update/urn:li:activity:7082363944891211776

**Comment by lbroudoux** [comment]
Please give us feedback on this! Is it useful? How do you expect it to help you? What are your ideas for other features? 
We are eager to hear from you!
